index.html

```diff
@@ -2595,6 +2595,135 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a <a>verifiable credential</a>, it is 
+          desirable to know whether the resource that is pointed to is the
+          same at signing time as it is at verification time.  This applies
+          to cases where there is an external resource that is
+          remotely retrieved as well as to cases where the <a>issuer</a> and/or
+          <a>verifier</a> may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) used in the <a>verifiable credential</a> are the same when
```

Comment:
can contexts be linked to a term definition?

Reply:
Perhaps putting `JSON-LD` in front of `context(s)` would be sufficient?

Reply:
```suggestion
          JSON-LD context(s) used in the <a>verifiable credential</a> are the same when
```

Reply:
`JSON-LD` was inserted, tho not using this change suggestion